include/sendemail.php

```diff
@@ -1083,8 +1083,18 @@ function generate_broken_build_message($emailtext, $Build, $Project)
     // Title
     $titleerrors .= '):';
     $title = 'FAILED ' . $titleerrors . ' ' . $Project->Name;
+    // In the sendmail.php file, in the sendmail function configure errors are now handled
+    // with their own logic, and the sendmail logic removes the 'configure_error' key therefore
+    // we should be able to verify that the following is a configure category by checking to see
+    // if the 'configure_error' key exists in the category array of keys.
+    $categories = array_keys($emailtext['category']);
 
-    if ($Build->GetSubProjectName()) {
+    $isConfErrorWithSubProjects = $Build->GetSubProjectName() &&
+        !in_array('configure_errors', $categories);
```

Comment:
This variable name is confusing.  It really represents "I have a subproject and this isn't a configure error". How about something like `$mentionSubProjectName` instead?

Reply:
Sounds good, I'll use that.